frontend/pages/schedule/ManageSchedulePage/components/ScheduleListWrapper/_styles.scss

```diff
@@ -0,0 +1,167 @@
+.schedule-list-wrapper {
+  border-collapse: collapse;
+  overflow: scroll;
```

Comment:
@RachelElysia I believe the `overflow: scroll;` is creating the funky scroll bars Zach is seeing. This is likely because he has an external mouse connected to his computer. Removing this property will do the trick (correct me if I'm wrong, but I don't think we need this).
```suggestion
```

Reply:
omg, ty!